examples/infiniband.yaml

```diff
@@ -0,0 +1,9 @@
+network:
+  version: 2
+  renderer: networkd
+  ethernets:
+    ib0:
+      match:
+        macaddress: "11:22:33:44:55:66:77:88:99:00:11:22:33:44:55:66:77:88:99:00"
```

Comment:
Is this the preferred format for representing infiniband mac addresses?  It doesn't seem very readable.  It does seem to be the format that NM uses, but are there alternatives in common use that we should consider?